server/src/main/java/org/elasticsearch/action/admin/indices/create/TransportCreateIndexAction.java

```diff
@@ -63,13 +73,48 @@ protected void masterOperation(Task task, final CreateIndexRequest request, fina
         }
 
         final String indexName = indexNameExpressionResolver.resolveDateMathExpression(request.index());
+
+        String mappings = request.mappings();
+        Settings settings = request.settings();
+        Set<Alias> aliases = request.aliases();
+
+        String concreteIndexName = indexName;
+        boolean isSystemIndex = false;
+
+        SystemIndexDescriptor descriptor = systemIndices.findMatchingDescriptor(indexName);
+
+        if (descriptor != null && descriptor.isAutomaticallyManaged()) {
+            isSystemIndex = true;
+            // System indices define their own settings and mappings, which cannot be overridden.
+            mappings = descriptor.getMappings();
+            settings = descriptor.getSettings();
+            concreteIndexName = descriptor.getPrimaryIndex();
+
+            if (descriptor.getAliasName() == null) {
+                aliases = Set.of();
+            } else {
+                aliases = Set.of(new Alias(descriptor.getAliasName()));
+            }
+        }
+
         final CreateIndexClusterStateUpdateRequest updateRequest =
-            new CreateIndexClusterStateUpdateRequest(cause, indexName, request.index())
+            new CreateIndexClusterStateUpdateRequest(cause, concreteIndexName, request.index())
                 .ackTimeout(request.timeout()).masterNodeTimeout(request.masterNodeTimeout())
-                .settings(request.settings()).mappings(request.mappings())
-                .aliases(request.aliases())
+                .aliases(aliases)
                 .waitForActiveShards(request.waitForActiveShards());
 
+        if (isSystemIndex) {
```

Comment:
Similar to the comment in `AutoCreateAction`, can we break up the system and non-system case since we don't really need to share anything between the two cases?